dialogue-target/src/main/java/com/palantir/dialogue/Request.java

```diff
@@ -82,6 +85,14 @@ private Request(Builder builder) {
         return body;
     }
 
+    /**
+     * The mutable request attachments for this request. Attachments will be propagated when this request is mutated
+     * through the builder
+     */
+    public RequestAttachments attachments() {
```

Comment:
May want to add a `@Beta` annotation here and on the `RequestAttachments` type for now.

Reply:
Ack